ingenialink/ethercat/servo.py

```diff
@@ -63,24 +63,66 @@ def _read_raw(  # type: ignore [override]
         self, reg: EthercatRegister, buffer_size: int = 0, complete_access: bool = False
     ) -> bytes:
         self._lock.acquire()
+        try:
+            value = self._read_raw_attempt(reg, buffer_size, complete_access)
+        except Emergency:
+            logger.info("An emergency message was received. Retrying read operation.")
```

Comment:
Why do you retry? If an error appears, it should be caught and raised as an ILError. What kind of errors are Emergency errors?

Reply:
Over voltages and so on. We handle these kinds of messages [differently](https://github.com/ingeniamc/ingeniamotion/blob/master/ingeniamotion/errors.py#L136). In any case, maybe what we should do is to handle them as is its done with [CANopen](https://github.com/ingeniamc/ingenialink-python/blob/master/ingenialink/canopen/servo.py#L114). For now, I think we should just ignore them as they interefe with the correct functioning of our code.